internal/quickstart/create_flag.go

```diff
@@ -67,6 +78,14 @@ func (m createFlagModel) View() string {
 	style := lipgloss.NewStyle().
 		MarginLeft(2)
 
+	if m.success {
+		successMessage := fmt.Sprintf("Flag %q created successfully!", m.flagKey)
```

Comment:
Should we use the flag name instead of the key since that's what the user typed? Otherwise they could be surprised if they type "my flag" and we show "my-flag." If we go that route it may be simpler to create a flag struct for the key and name instead of passing both around.